.github/workflows/breaking-change-code.yaml

```
@@ -28,12 +28,13 @@ jobs:
         id: swagger-breaking-change-analyze-code
         run: |
           npm exec --no -- openapi-diff-runner \
-            --srp "${{ github.workspace }}" \
-            --number "${{ github.event.pull_request.number }}" \
-            --sb "${{ github.event.pull_request.head.ref }}" \
-            --tb "${{ github.event.pull_request.base.ref }}" \
-            --hc "${{ github.event.pull_request.head.sha }}" \
-            --repo "${{ github.repository }}"
+            -p "${{ github.workspace }}" \
```

```suggestion
            -p "$GITHUB_WORKSPACE" \
```

We might have this same issue in other workflows, but I think env vars are the safest at preventing script injections.  All params should use env vars, either built-in or custom-defined.

---

In this specific case, it doesn't secure more by using `GITHUB_WORKSPACE` since github context is also controlled by the GitHub action and it doesn't have direct user input.